src/ir_Delonghi.h

```diff
@@ -22,79 +22,49 @@
+    uint8_t Temp      :5;
+    uint8_t Fan       :2;
+    uint8_t Fahrenheit:1;
+    uint8_t Power     :1;
+    uint8_t Mode      :3;
+    uint8_t Boost     :1;
+    uint8_t Sleep     :1;
+    uint8_t           :2;
+    uint8_t OnTimer   :1;
+    uint8_t OnHours   :5;
+    uint8_t           :2;
+    uint8_t OnMins    :6;
+    uint8_t           :2;
+    uint8_t OffTimer  :1;
+    uint8_t OffHours  :5;
+    uint8_t           :2;
+    uint8_t OffMins   :6;
+    uint8_t           :2;
+    uint8_t Sum       :8;
+  };
+};
 
 // Constants
-const uint8_t kDelonghiAcTempOffset = 8;
-const uint8_t kDelonghiAcTempSize = 5;
 const uint8_t kDelonghiAcTempMinC = 18;  // Deg C
 const uint8_t kDelonghiAcTempMaxC = 32;  // Deg C
 const uint8_t kDelonghiAcTempMinF = 64;  // Deg F
 const uint8_t kDelonghiAcTempMaxF = 90;  // Deg F
 const uint8_t kDelonghiAcTempAutoDryMode = 0;
 const uint8_t kDelonghiAcTempFanMode = 0b00110;
-const uint8_t kDelonghiAcFanOffset = kDelonghiAcTempOffset +
-    kDelonghiAcTempSize;  // 13
-const uint8_t kDelonghiAcFanSize = 2;
 const uint8_t kDelonghiAcFanAuto =   0b00;
 const uint8_t kDelonghiAcFanHigh =   0b01;
 const uint8_t kDelonghiAcFanMedium = 0b10;
 const uint8_t kDelonghiAcFanLow =    0b11;
-const uint8_t kDelonghiAcTempUnitBit = kDelonghiAcFanOffset +
-    kDelonghiAcFanSize;  // 15 (1 = Celsius, 0 = Fahrenheit)
-const uint8_t kDelonghiAcPowerBit = kDelonghiAcTempUnitBit + 1;  // 16
-const uint8_t kDelonghiAcModeOffset = kDelonghiAcPowerBit + 1;  // 17
-const uint8_t kDelonghiAcModeSize = 3;
 const uint8_t kDelonghiAcCool =   0b000;
 const uint8_t kDelonghiAcDry =    0b001;
 const uint8_t kDelonghiAcFan =    0b010;
 const uint8_t kDelonghiAcAuto =   0b100;
-const uint8_t kDelonghiAcBoostBit = kDelonghiAcModeOffset +
-    kDelonghiAcModeSize;  // 20 (Aka Turbo)
-const uint8_t kDelonghiAcSleepBit = kDelonghiAcBoostBit + 1;  // 21
-// Two zero bits
-const uint8_t kDelonghiAcOnTimerEnableBit = kDelonghiAcSleepBit + 3;  // 24
-const uint8_t kDelonghiAcHoursSize = 5;  // Max 23 hrs
-const uint8_t kDelonghiAcMinsSize = 6;  // Max 59 mins
 const uint16_t kDelonghiAcTimerMax = 23 * 60 + 59;
-const uint8_t kDelonghiAcOnTimerHoursOffset = kDelonghiAcOnTimerEnableBit +
-    1;  // 25
-const uint8_t kDelonghiAcOnTimerMinsOffset = kDelonghiAcOnTimerHoursOffset +
-    kDelonghiAcHoursSize + 2;  // 32  (inc another two zero bits)
-// Two zero bits
-const uint8_t kDelonghiAcOffTimerEnableBit = kDelonghiAcOnTimerMinsOffset +
-    kDelonghiAcMinsSize + 2;  // 40
-const uint8_t kDelonghiAcOffTimerHoursOffset = kDelonghiAcOffTimerEnableBit +
-    1;  // 41
-const uint8_t kDelonghiAcOffTimerMinsOffset = kDelonghiAcOffTimerHoursOffset +
-    kDelonghiAcHoursSize + 2;  // 48  (inc another two zero bits)
-// Two zero bits
-const uint8_t kDelonghiAcChecksumOffset = kDelonghiAcOffTimerMinsOffset +
-    kDelonghiAcMinsSize + 2;  // 56
-const uint8_t kDelonghiAcChecksumSize = 8;
-
+const uint8_t kDelonghiAcChecksumOffset = 56;
```

Comment:
Is this needed? I can't see it being used anymore.

Reply:
in IRDelonghiAc::calcChecksum, ir_Delonghi.cpp, line 114

Reply:
Oh cool. I missed it there. Thanks.